src/commands/index.ts

```diff
@@ -2,17 +2,18 @@ import * as vscode from "vscode";
 
 import { RequestError } from "@octokit/request-error";
 
+import * as cp from "child_process";
```

Comment:
This import broke the web build.

Reply:
Sorry, I don't know what you mean. It would be great if you could give an error message.

Reply:
I believe there is no `child-process` lib in browser

https://github.com/ultram4rine/vscode-choosealicense/actions/runs/5097303506/jobs/9165152836#step:7:15

```sh
$ esbuild ./src/extension.ts --bundle --external:vscode --tsconfig=./tsconfig.json --log-level=info --color=true --format=cjs --minify --outfile=./out/web/extension.js
✘ [ERROR] Could not resolve "child_process"

    src/commands/index.ts:5:20:
      5 │ import * as cp from "child_process";
        ╵                     ~~~~~~~~~~~~~~~

  The package "child_process" wasn't found on the file system but is built into node. Are you trying to bundle for node? You can use "--platform=node" to do that, which will remove this error.
```

Reply:
Like it say that, you can try to add `--platform=node` to build it, this way is easy to use.
refer https://esbuild.github.io/getting-started/#bundling-for-node

Reply:
Than it will create node bundle that will not work in browser 😕

Reply:
I google a lot, but I didn't find a way to use `child_process` in the browser. 😭

Reply:
Did you check the `git` vscode extension (https://github.com/microsoft/vscode/blob/eef30e7165e19b33daa1e15e92fa34ff4a5df0d3/extensions/git/src/api/git.d.ts#L183) as I mentioned here https://github.com/ultram4rine/vscode-choosealicense/pull/786#issuecomment-1565350013?

Reply:
I have refactored code, remember to check it.

Reply:
@whosydd this works, can I push this to your repo?
```js
    if (author) {
      text = replaceAuthor(author, license.key, text);
    } else {
      const gitExtension =
        vscode.extensions.getExtension<GitExtension>("vscode.git")?.exports;
      if (gitExtension) {
        const git = gitExtension.getAPI(1);
        const gitUsername = await git.repositories[0].getGlobalConfig(
          "user.name"
        );

        if (gitUsername !== "") {
          text = replaceAuthor(gitUsername, license.key, text);
        }
      }
    }
```

Reply:
I'm ok with it.